lib/active_record/connection_adapters/cockroachdb/schema_statements.rb

```diff
@@ -32,6 +32,63 @@ def primary_key(table_name)
           end
         end
 
+        # OVERRIDE: Added `unique_rowid` to the last line of the second query.
+        #   This is a CockroachDB-specific function used for primary keys.
+        #
+        # Returns a table's primary key and belonging sequence.
+        def pk_and_sequence_for(table) # :nodoc:
+          # First try looking for a sequence with a dependency on the
+          # given table's primary key.
+          result = query(<<~SQL, "SCHEMA")[0]
+            SELECT attr.attname, nsp.nspname, seq.relname
```

Comment:
@rafiss does this make sense for CRDB ? All the tests I'm working with until now are giving a void result...

Reply:
yes, this looks correct

if you are getting no results, it's probably because CRDB defaults to not using sequences for SERIAL types. this can be changed with the serial_normalization setting: https://www.cockroachlabs.com/docs/stable/serial#modes-of-operation